README.md

```diff
@@ -1 +1,33 @@
-# tf_cloudtrail
\ No newline at end of file
+# tf_cloudtrail
+
+Setup and manage CloudTrail
+
+### Variables
+
+#### Required
+
+* name: Name of CloudTrail trail.
+* region: AWS region. Used to find remote state.
+
+
+
+#### Optional
+
+* enable_logging: Enable logging, set to 'false' to Pause logging (default = true)
```

Comment:
Use markdown formatting for code. 
E.g. `enable_logging`

@comeanother please help format explanation.